### DISCUSSION TITLE
Chinese-Alpaca-2-7B微调步骤

### DISCUSSION THREAD

**Discussion by Olivia-xu** [ymcui/Chinese-LLaMA-Alpaca-2#113] [Q&A]
请教下楼主，Chinese-Alpaca-2-7B是基于什么模型微调过来的，可不可以开放一下微调步骤

**Comment by airaria** [comment]
基于Chinese-LLaMA-2微调的：
https://github.com/ymcui/Chinese-LLaMA-Alpaca-2/wiki/sft_scripts_zh
